indico/modules/categories/client/js/components/CategoryStatistics.jsx

```diff
@@ -0,0 +1,196 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import categoryStatisticsURL from 'indico-url:categories.statistics_json';
+
+import moment from 'moment';
+import React, {useMemo} from 'react';
+import PropTypes from 'prop-types';
+import {Chart} from 'react-charts';
+import {Container, Header, Segment, Statistic, Message, Loader} from 'semantic-ui-react';
+import {setMomentLocale} from 'indico/utils/date';
+import {Translate} from 'indico/react/i18n';
+import {useIndicoAxios} from 'indico/react/hooks';
+
+import './CategoryStatistics.module.scss';
+
+export default function CategoryStatistics({categoryId, lang}) {
+  const {data, loading: isLoading} = useIndicoAxios({
+    url: categoryStatisticsURL({category_id: categoryId}),
+    camelize: true,
+    trigger: categoryId,
+  });
+
+  if (isLoading) {
+    return <Loader size="massive" active />;
+  } else if (!data) {
+    return null;
+  }
+
+  return <CategoryStatisticsDisplay data={data} lang={lang} />;
+}
+
+CategoryStatistics.propTypes = {
+  categoryId: PropTypes.number.isRequired,
+  lang: PropTypes.string.isRequired,
+};
+
+function getGraphData(data) {
+  const serialize = statsData =>
+    Object.entries(statsData)
+      .filter(([year]) => year <= data.maxYear + 3 && year >= data.minYear)
+      .map(([year, number]) => [moment(year, 'YYYY'), number]);
+
+  const series = {type: 'bar'};
+  const tooltip = {align: 'left', anchor: 'left'};
+  const axes = [
+    {primary: true, type: 'time', position: 'bottom'},
+    {type: 'linear', position: 'left'},
+  ];
+  const eventGraph = [
+    {
+      label: Translate.string('Events'),
+      data: serialize(data.events),
+    },
+  ];
+  const contributionsGraph = [
+    {
+      label: Translate.string('Contributions'),
+      data: serialize(data.contributions),
+    },
+  ];
+
+  return {eventGraph, contributionsGraph, series, axes, tooltip};
+}
+
+function CategoryStatisticsDisplay({data, lang}) {
+  const userLanguage = lang.replace('_', '-');
+  setMomentLocale(lang);
```

Comment:
I would move this `setMomentLocale` to `categories/index.js` since this is configuration which should happen before mounting the component. This also actually fixes `moment.utc(data.updated).format('LLL');` since right now `updatedTime` isn't properly formatted using the current locale.

Reply:
Sounds good to me, I will update this :+1: